### ISSUE TITLE
Cannot translate the value of keypath 'message.hello'. Use the value of keypath as default.

### ISSUE THREAD

**Issue by Konafets** [martinlindhe/laravel-vue-i18n-generator#29]
I encountered that error above and read all issue reports at [kazupon/vue-i18n](https://github.com/kazupon/vue-i18n) repo. The error happens apparently because the Vue instance is loaded before the localization.

One guy pointed to [vuex-i18n](https://github.com/dkfbasel/vuex-i18n), another translation lib. I gave it a shot and this worked like a charm. The best thing: I could use the generated localization files from this package with [vuex-i18n](https://github.com/dkfbasel/vuex-i18n). Here is my setup:

```js
import store from './vuex';
import vuexI18n from 'vuex-i18n';
Vue.use(vuexI18n.plugin, store);

import Locales from './vue-i18n-locales.generated.js';
Vue.i18n.add('en', Locales.en);
Vue.i18n.add('de', Locales.de);

// set the start locale to use
Vue.i18n.set(Laravel.locale);

const app = new Vue({
    store, // inject store into all children
    el: '#app',
});
```

The `Laravel.locale` comes from my Layout:

```html
<head>
    <!-- Header stuff -->
  
    <!-- Scripts -->
    <script>
        const Laravel = {
            csrfToken: "{{ csrf_token() }}",
            locale: "{{ config('app.locale') }}"
        };
    </script>
</head>
```

**Comment by martinlindhe** [comment]
Nice find, we should update the README with info about this!

**Comment by Abonive** [comment]
So, I got around this error, like this.

```
import Vue from 'vue'
import VueI18n from 'vue-i18n'
import Locales from './vue-i18n-locales.generated.js'

Vue.use(VueI18n)

const i18n = new VueI18n({locale: 'en', messages: Locales})

const app = new Vue({
  i18n,
  el: '#app',
})
```

I've tried changing the language to other than English, selecting the Vue root instance in devtools and running

`$vm0.$i18n.locale = 'es'`

and it works fine.

**Comment by martinlindhe** [comment]
@

**Comment by martinlindhe** [comment]
@Abonive Something similar is already documented in the [README](https://github.com/martinlindhe/laravel-vue-i18n-generator#vue-i18n) (a default language needs to be set):

```js
import Vue from 'vue';
import VueInternalization from 'vue-i18n';
import Locales from './vue-i18n-locales.generated.js';

Vue.use(VueInternalization);

Vue.config.lang = 'en';

...
```


Also gonna close this, as documented solutions is implemented in #31 and #34
